x/session/keeper/keeper.go

```diff
@@ -60,12 +60,11 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 func (k Keeper) BeginBlocker(ctx sdk.Context) {
 	// ctx.BlockHeader().LastBlockId.Hash is the hash of the last block committed
 	hash := ctx.BlockHeader().LastBlockId.Hash
-	// ctx.BlockHeader().Height is the height of the current block being processed,
-	// decrementing it by 1 gives us the height of the last block committed.
-	height := ctx.BlockHeader().Height - 1
-	// If height is 0, ctx.BlockHeader().LastBlockId.Hash will be nil,
+	// ctx.BlockHeader().Height is the height of the last committed block.
+	height := ctx.BlockHeader().Height
```

Comment:
@red-0ne Can you explain what we missed or misunderstood in our last review cycle? I want to understand why it got through code review.

<img width="827" alt="Screenshot 2024-02-06 at 9 26 08 PM" src="https://github.com/pokt-network/poktroll/assets/1892194/1da9c9a3-6d1c-438c-8bbb-bd7566cc881a">

Reply:
Can you #PUC to a link that proves `ctx.BlockHeader().Height is the height of the last committed block.`

Reply:
This has been confirmed by comparing the block heights and hashes we got from the CometBFT websocket client and the values of `ctx.BlockHeader().Height` and `ctx.BlockHeader().LastBlockId.Hash`.

I couldn't find any statement of that in the CosmosSDK or code. The closest I got was this reference:

The only relevant info I found was this https://github.com/cosmos/cosmos-sdk/blob/main/baseapp/grpcserver.go#L60